src/hw9_Tasks/ContainsDuplicate.java

```diff
@@ -0,0 +1,37 @@
+package hw9_Tasks;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public class ContainsDuplicate {
+    public static void main(String[] args) {
+        System.out.println(containsDuplicateStream(new int[]{1,2,3,1}));
+        System.out.println(containsDuplicateArray(new int[]{1,2,3,1}));
+        System.out.println(containsDuplicateSet(new int[]{1,2,3,1}));
+    }
+
+    // 27 ms
+    public static boolean containsDuplicateStream(int[] nums) {
+        return Arrays.compare(nums, Arrays.stream(nums).distinct().toArray()) != 0;
+    }
+
+    // 20 ms
+    public static boolean containsDuplicateArray(int[] nums) {
+        Arrays.sort(nums);
+        for (int i = 0; i < nums.length - 1; i++) {
+            if (nums[i] == nums[i + 1]) return true;
+        }
+        return false;
+    }
+
+    // 9 ms
+    public static boolean containsDuplicateSet(int[] nums) {
+        Set<Integer> uniqueNums = new HashSet<>();
+        for (Integer i : nums) {
+            if (uniqueNums.contains(i)) return true;
+            uniqueNums.add(i);
+        }
+        return false;
```

Comment:
Very goo solution, but what do you think about size of the arrays? Will the it be different from the size of the set in case of duplicates?

Reply:
Yes, i think in case of duplicates, the size of the set will be different from the size of given array, because we add new elements to set one by one. When we find a duplicate, we stop the method and don't add new element.